test/test_dispatch.py

```diff
@@ -374,6 +374,114 @@ def test_computed_table_with_cpu_catchall(self):
 AutogradCPU: fallthrough registered at /dev/null:0 [backend fallback]
 AutogradCUDA: default_def_name_t_t [catch all]
 AutogradXLA: default_def_name_t_t [catch all]
+''')
+
+    def test_computed_table_with_math(self):
+        global_m = C._dispatch_library("IMPL", "_", "autogradcpu")
+        result = self.commute("foo", [
+            # m.def("foo", [](const Tensor & x) { return x })
+            lambda m: m.def_name_t_t("foo"),
```

Comment:
OK, remind me: in the glorious new future, what key does `m.def("foo", foo_impl);` get registered to? It's going to be Math, right? So this test isn't very future compatible, because what you have here is an implicit definition of math here, and then an explicit definition of math below. When you eventually cut over, this test should start failing as duplicate definitions of the same type will be non-commutative.

I think probably what you intended here to do is to only `def_name` with out the actual default definition. That would give you give you a computed table with only a math entry, which sounds like what you implied in your test name.

Reply:
It looks like something similar happened for all the tests below.